clients/iscmove/iscmoveclient/client_request_test.go

```diff
@@ -125,8 +126,9 @@ func TestCreateAndSendRequest(t *testing.T) {
 	for range 25 + 26 {
 		coinRef, _ := buildDeployMintTestcoin(t, client, cryptolibSigner)
 		testCoinRef = append(testCoinRef, coinRef)
+		time.Sleep(3 * time.Second)
```

Comment:
Isn't it possible to avoid adding sleeps? Or at least reducing the amount of time, 3s is an eternity :stuck_out_tongue: